README.md

```diff
@@ -115,6 +115,10 @@ $ npm run deploy
 #  deploy:<platform>-<architecture>
 # example:
 $ npm run deploy:darwin-x64
+
+# If you have fpm installed, you can also build RPM and Deb packages:
```

Comment:
add link here to fpm repo and maybe between parenthesis the install instructions `gem install fpm`